### ISSUE TITLE
Add alpha integration to MarkerView

### ISSUE THREAD

**Issue by tobrun** [mapbox/mapbox-gl-native#5080]
Google exposes customization of **alpha**:

> Sets the opacity of the marker. Defaults to 1.0.

Now that #3276 landed we can implement this in MarkerView/MarkerViewOptions.

**Comment by tobrun** [comment]
![ezgif com-video-to-gif 18](https://cloud.githubusercontent.com/assets/2151639/15468344/176eaa6a-20e4-11e6-8482-b2a2bbcd7dd1.gif)

with related test code:

```java
    private boolean opaque;

    private void animateAlpha(final MarkerView markerView) {
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                markerView.setAlpha(opaque ? 0.2f : 1.0f);
                
                opaque = !opaque;
                animateAlpha(markerView);
            }
        }, 3000);
    }
```

**Comment by tobrun** [comment]
Landed with 40bb02fe749ea8221a8da57b715bcbc1866edb8b

**Comment by basitsaleemmb** [comment]
@tobrun 
how can i fade out a marker rather than a markerView in mapBoxMap
